dashboard/app/models/pd/workshop_daily_survey.rb

```diff
@@ -21,5 +21,42 @@
 #  index_pd_workshop_daily_surveys_on_user_workshop_day_form  (user_id,pd_workshop_id,day,form_id) UNIQUE
 #
 
-class Pd::WorkshopDailySurvey < ActiveRecord::Base
+module Pd
+  class WorkshopDailySurvey < ActiveRecord::Base
+    include JotFormBackedForm
+
+    belongs_to :user
+    belongs_to :pd_session, class_name: 'Pd::Session'
+    belongs_to :pd_workshop, class_name: 'Pd::Workshop'
+
+    # @override
+    def self.attribute_mapping
+      {
+        user_id: 'userId',
+        pd_session_id: 'sessionId',
+        pd_workshop_id: 'workshopId',
+        day: 'day'
+      }
+    end
+
+    validates_presence_of(
+      :user_id,
+      :pd_workshop_id,
+      :day
+    )
+
+    VALID_DAYS = (0..5).freeze
```

Comment:
Is there a post workshop survey as well?

Reply:
I'm considering it day 5 for now. We can adjust if that changes.